src/parse.js

```diff
@@ -102,17 +103,24 @@ export default function parse (str, {meta} = {}) {
 							Object.assign(meta, {formatId: "color", types});
 						}
 
+						if (colorSpec.id.startsWith("--") && !id.startsWith("--")) {
+							console.warn(`Use prefixed color(${colorSpec.id}) instead of color(${id}).`);
```

Comment:
We need to clarify *why* this distinction exists, otherwise the warning makes it sound like Color.js is annoying for no reason. E.g. something like:

```suggestion
							console.warn(`${space.name} is a non-standard space, so you should use color(${colorSpec.id}) instead of color(${id}).`);
```

Reply:
@LeaVerou Adjusted in https://github.com/color-js/color.js/pull/439/commits/b1cf95a4a68dd7c54f0a6e62609318be780e9d92 -- what do you think?